.factory/FACTORY.md

```
@@ -0,0 +1,124 @@
+proposals: on
+
+<!-- May the loops file `bot:idea` issues of their own (tech debt they keep hitting,
+     follow-ups a PR surfaced)? Proposals arrive already in build-order form and are NEVER
+     promoted by the bot — only a maintainer applies bot:build. Set `proposals: off` to
+     disable; note that on a public repo, bot-filed issues are visible to everyone. -->
+
+## Local gates
+
+The command chain that must be green before any PR flips ready:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)